test/integration/aens.ts

```diff
@@ -1,6 +1,6 @@
 /*
```

Comment:
I believe you have an error in your `tsconfig.json`  file.
`"include": ["src/**/*", "ts/integration/**/*"]`
should be: 
`"include": ["src/**/*", "test/integration/**/*", "test/unit/**/*"]`
_test/integration, not ts/integration_
this means that your test file didn't properly flag errors. If you change the config file you'll get a few more errors other than the ones mentioned in the review.